utils/http.py

```diff
@@ -2,14 +2,24 @@
 
 import requests
 from PIL import Image
+import json
+
+config = json.load(open('config.json'))
 
 
 def get_image(url):
     try:
-        return Image.open(BytesIO(requests.get(url, stream=True).content))
+        if 'proxy_url' in config:
+            return Image.open(BytesIO(requests.get(config['proxy_url'] + '?url=' + url,

```

Comment:
I think it's better to use the in-built GET parameter generator in requests. `requests.get(config['proxy_url'], params={'url': url})`